docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md

```diff
@@ -232,7 +232,19 @@ This property is only valid for tests run against emulators. It has no effect on
 sauce:
   visibility: private
 ```
+---
+
+### `launchOrder`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Control starting order of suites. If unspecified or empty, the default is the order in which suites are written in the config file.
```

Comment:
```suggestion
Specifies the execution order for your test suites. When set to `fail rate`, test suites with the highest failure rate will execute first. If unspecified, test suites will execute in the order in which they are written in the configuration file. 
```

Reply:
@tianfeng92 I added a couple suggested changes to merge the two lines into one -- the suggestions apply to all files.

Reply:
🙆‍♀️

Reply:
@tianfeng92 Thank you!